includes/footer.html

```diff
@@ -60,7 +60,12 @@ <h5 class="white-text fine">coala</h5>
     </div>
     <div class="footer-copyright">
         <div class="container">
-            <span class="left">© 2017 coala</span>
+            <span class="left">© 
+                <script>
+                    document.write(new Date().getFullYear());
+                </script> 
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpr9xhq323/includes/footer.html
+++ b/tmp/tmpr9xhq323/includes/footer.html
@@ -63,7 +63,7 @@
             <span class="left">© 
                 <script>
                     document.write(new Date().getFullYear());
-                </script> 
+                </script>
                 coala
             </span>
             <a class="grey-text text-lighten-4 right" href="#!">GNU AGPL v3.0</a>
```